include/linux/ipts-companion.h

```diff
@@ -0,0 +1,22 @@
+#ifndef _INTEL_IPTS_COMPANION_H_
+#define _INTEL_IPTS_COMPANION_H_
+
+#include <linux/firmware.h>
+#include <linux/ipts-binary.h>
+
+typedef struct ipts_companion ipts_companion_t;
+
+typedef int (*ipts_fw_handler_t)(const struct firmware **, const char *,
+		struct device *, ipts_companion_t *companion);
```

Comment:
Since we're going over this again, I'd kind of prefer having the `companion` as the first parameter. Makes the association a bit clearer (as in "for companion x load firmware with parameters y, z, ...").

Reply:
Done.